### ISSUE TITLE
Search filters should be interact-able with "enter"

### ISSUE THREAD

**Issue by SMaxOwok** [ManifoldScholar/manifold#2008]
>Combo box for search filters only expands on spacebar keypress, not enter key

Enter key is submitting search form.  We need to better scope the event.

**Comment by SMaxOwok** [comment]
Upon further investigation, select boxes are not expected to react to `enter`.  To improve this would mean refactoring to a custom "select-like" element with the listbox role.

https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html#sc_label

**Comment by SMaxOwok** [comment]
Alternatively we could adjust our selects to respond to `enter`.  I'm closing this for now because I think our select filters are accessible in that they can still be triggered by pressing `space` and are fully interact-able.
